code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -144,14 +144,14 @@
 	embed_type = /datum/embedding/bullet/flechette
 
 /datum/embedding/bullet/flechette
-	embed_chance = 100
-	fall_chance = 0
+	embed_chance = 25
+	fall_chance = 10
 	jostle_chance = 20
 	ignore_throwspeed_threshold = TRUE
 	pain_stam_pct = 0.1
 	pain_mult = 0.5
 	jostle_pain_mult = 1.5
-	rip_time = 1 SECONDS
+	rip_time = 0.5 SECONDS
```

Comment:
Bullet embeds are, by default, hidden and thus can't be ripped out by hand - only wirecutter and hemostat. Meaning this doesn't (ultimately) change much.

Just so you're aware.

Reply:
Mmm I'll change the PR body a bit in that case.